apps/client/src/app/implementation-guide-wrapper/r4/resource-modal.component.html

```diff
@@ -11,14 +11,6 @@ <h4 class="modal-title">Edit Resource</h4>
   <app-fhir-string [parentObject]="resource" propertyName="name" title="Name" [required]="true" tooltipPath="ImplementationGuide.definition.resource.name"></app-fhir-string>
   <span *ngIf="!resource.name">You must specify a name.</span>
 
-  <div class="form-group">
```

Comment:
Let's leave the description field in the pop-up window, even though it's already shown in the table. Would like the pop-up window to represent _everything_ related to a resource, while the table includes only the most common of things that require editing in bulk.